packages/terser/test/inline_sourcemap/spec.js

```diff
@@ -14,4 +15,19 @@ describe('terser sourcemap handling', () => {
       expect(pos.column).toBe(14);
     });
   });
+
+  it('should link the source to the source map', () => {
+    const file = require.resolve(__dirname + '/out.min.js');
```

Comment:
Because __dirname is an absolute path `require.resolve` can be replaced with `path.join`
This is helpful because when we look at require.resolve use to refactor out loader.js logic there will be fewer of them to understand.